templates/kernel/lifecycle.html

```diff
@@ -0,0 +1,784 @@
+            <tr>
+              <td>&nbsp;</td>
+              <th scope="col">Released</th>
+              <th scope="col">End of Life</th>
+              <th scope="col">Extended security maintenance</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><strong>Ubuntu 14.04 LTS (v3.13)</strong></td>
+              <td>Apr 2014</td>
+              <td>Apr 2019</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 16.04 LTS (v4.4)</strong></td>
+              <td>Apr 2016</td>
+              <td>Apr 2021</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 18.04 LTS (v4.15)</strong></td>
+              <td>Apr 2018</td>
+              <td>Apr 2023</td>
+            </tr>
+            <tr>
+              <td>Ubuntu 19.04 (v5.0)</td>
+              <td>Apr 2019</td>
+              <td>Jan 2020</td>
+            </tr>
+            <tr>
+              <td>Ubuntu 19.10 (v5.3)</td>
+              <td>Oct 2019</td>
+              <td>Jul 2020</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 20.04 LTS</strong></td>
+              <td>Apr 2020</td>
+              <td>Apr 2025</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip is-bordered">
+    <div class="row">
+      <div class="col-8">
+        <h3>Kernel support schedule</h3>
+
+        <nav class="p-tabs">
+          <ul class="p-tabs__list" role="tablist">
+            <li class="p-tabs__item" role="presentation">
+              <a href="#support-all" class="p-tabs__link" role="tab" id="support-all-tab" aria-controls="support-all" aria-selected="true">All releases</a>
+            </li>
+            <li class="p-tabs__item" role="presentation">
+              <a href="#support-lts" class="p-tabs__link" role="tab" id="support-lts-tab" aria-controls="support-lts">All LTS release</a>
+            </li>
+            <li class="p-tabs__item" role="presentation">
+              <a href="#support-20-04" class="p-tabs__link" role="tab" id="support-20-04-tab" aria-controls="support-20-04">20.04 LTS</a>
+            </li>
+            <li class="p-tabs__item" role="presentation">
+              <a href="#support-18-04" class="p-tabs__link" role="tab" id="support-18-04-tab" aria-controls="support-18-04">18.04 LTS</a>
+            </li>
+            <li class="p-tabs__item" role="presentation">
+              <a href="#support-16-04" class="p-tabs__link" role="tab" id="support-16-04-tab" aria-controls="support-16-04">16.04 LTS</a>
+            </li>
+            <li class="p-tabs__item" role="presentation">
+              <a href="#support-14-04" class="p-tabs__link" role="tab" id="support-14-04-tab" aria-controls="support-14-04">14.04 LTS</a>
+            </li>
+          </ul>
+        </nav>
+
+        <div class="p-tabs__content" id="support-20-04" role="tabpanel" aria-labelledby="installation-20-04-tab">
```

Comment:
This should be `aria-labelledby="support-20-04-tab"` and the same for the others.